content/src/components.ts

```diff
@@ -76,7 +76,7 @@ export async function initComponentsWithEnv(env: Environment): Promise<AppCompon
 
   const catalystFetcher = FetcherFactory.create({ env })
   const contentFolder = path.join(env.getConfig(EnvironmentConfig.STORAGE_ROOT_FOLDER), 'contents')
-  const storage = await createFileSystemContentStorage({ fs }, contentFolder)
+  const storage = await createFolderBasedFileSystemContentStorage({ fs }, contentFolder)
```

Comment:
Could we move the `fs` interface + implementation to `catalyst-storage` too? (Or remove it within `catalyst-storage`, so it's not duplicated)

Reply:
Yes, will do

Reply:
👍